content/python/concepts/math-module/terms/math-isfinite/math-isfinite.md

```diff
@@ -0,0 +1,56 @@
+---
+Title: 'math.isfinite()'
+Description: 'Returns True when a number is finite or False.'
+Subjects:
+ - 'Computer Science'
+ - 'Data Science'
+Tags:
+ - 'Arithmetic'
+ - 'Boolean'
+ - 'Functions'
+CatalogContent:
+ - 'learn-python-3'
+ - 'paths/computer-science'
+---
+
+The **`math.isfinite()`** function returns true when a number is finite or false. The number should neither be an infinite nor NaN. 
```

Comment:
```suggestion
The **`math.isfinite()`** function returns `True` when a number is finite and `False` otherwise. The number should neither be infinite nor `NaN`.
```